File: Frontend/lib/model/learner.dart
class Learner {
  final int id;
  final String firstName;
  final String lastName;
  final String birthdate;
  final String age;
  final String gender;
  final String comesFrom;
  final String description;

  Learner({
    required this.id,
    required this.firstName,
    required this.lastName,
    required this.birthdate,
    required this.age,
    required this.gender,
    required this.comesFrom,
    required this.description,
  });

  factory Learner.fromJson(Map<String, dynamic> json) {
    return Learner(
      id: json["id"],
      firstName: json["firstName"],
      lastName: json["lastName"],
      birthdate: json["birthdate"],
      age: json["age"],
      gender: json["gender"],
      comesFrom: json["comesFrom"],
      description: json["description"],
    );
  }

  Map toJson() {
    return {
      "id": id,
      "firstName": firstName,
      "lastName": lastName,
      "birthdate": birthdate,
      "age": age,
      "gender": gender,
      "comesFrom": comesFrom,
      "description": description,
    };
  }
}

File: Backend/Models/Entities/Learner.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace InternshippClass.Models.Entities
{
    public class Learner
    {
        public Learner()
        {
            Marks = new HashSet<Mark>();
        }
        public long Id { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public DateTime BirthDate { get; set; }

        public int Age { get; set; }

        public string Gender { get; set; }

        public string Institution { get; set; }

        public string Description { get; set; }

        public long MarkId { get; set; }

        public virtual ICollection<Mark> Marks { get; set; }
    }
}
